pkg/graph/schema.graphql

```diff
@@ -563,6 +572,8 @@ type Mutation {
     @hasRole(role: EASI_GOVTEAM)
   updateTestDate(input: UpdateTestDateInput!): UpdateTestDatePayload
     @hasRole(role: EASI_508_TESTER)
+  removeTestDate(input: RemoveTestDateInput!): RemoveTestDatePayload
```

Comment:
Isaac is using "delete" rather than "remove" in his PR. I think it would be nice to have some consistency in our language.

Reply:
Im fine with changing mine